general/administration/Hardware-Acceleration/ubuntu-debian-nvenc.md

```diff
@@ -0,0 +1,161 @@
+                00:17.0 SATA controller: Intel Corporation 200 Series PCH SATA controller [AHCI mode]
+                00:1f.0 ISA bridge: Intel Corporation Device a2ca
+                00:1f.2 Memory controller: Intel Corporation 200 Series/Z370 Chipset Family Power Management Controller
+                00:1f.3 Audio device: Intel Corporation 200 Series PCH HD Audio
+                00:1f.4 SMBus: Intel Corporation 200 Series/Z370 Chipset Family SMBus Controller
+                00:1f.6 Ethernet controller: Intel Corporation Ethernet Connection (2) I219-V
+                01:00.0 VGA compatible controller: NVIDIA Corporation TU116 [GeForce GTX 1650 SUPER] (rev a1)
+                01:00.1 Audio device: NVIDIA Corporation TU116 High Definition Audio Controller (rev a1)
+                01:00.2 USB controller: NVIDIA Corporation TU116 USB 3.1 Host Controller (rev a1)
+                01:00.3 Serial bus controller [0c80]: NVIDIA Corporation TU116 [GeForce GTX 1650 SUPER] (rev a1)`
+        
+        - This is the ouput you are looking for
+
+        01:00.3 Serial bus controller [0c80]: NVIDIA Corporation TU116 [GeForce GTX 1650 SUPER] (rev a1)
+        Now check the firmware revision its on by typing 
+
+                ndvidia-smi
+
+        - you should get an ouput like this
+
+                Sun Aug 16 20:42:25 2020
+                +-----------------------------------------------------------------------------+
+                | NVIDIA-SMI 450.57       Driver Version: 450.57       CUDA Version: 11.0     |
+                |-------------------------------+----------------------+----------------------+
+                | GPU  Name        Persistence-M| Bus-Id        Disp.A | Volatile Uncorr. ECC |
+                | Fan  Temp  Perf  Pwr:Usage/Cap|         Memory-Usage | GPU-Util  Compute M. |
+                |                               |                      |               MIG M. |
+                |===============================+======================+======================|
+                |   0  GeForce GTX 165...  Off  | 00000000:01:00.0  On |                  N/A |
+                |  0%   50C    P0    26W / 100W |    719MiB /  3895MiB |     17%      Default |
+                |                               |                      |                  N/A |
+                +-------------------------------+----------------------+----------------------+
+
+   2. So to find the correct drivers select from the dropdown. Againb using my Geoforce 970 as an example
+
+        Product Type: Geoforce (whoever manufacters your GPU)
+        Product Series: Geoforce 900 Series (make sure you are not selecting the M unless your on a laptop)
+        Product Geoforce GTX 970 (Select what model of graphics card you have for your GPU)
+        Operating System: LINUX 64BIT        
+        Download Type: 
+        
+        **(Important)**  Select Linux Short Lived if your graphics card is a bit older.WHEN I SAY OLDER I MEAN 2 YEARS OR LESS.
```

Comment:
Should this be a fixed year (like 2018 in this case) or always 2 years, relative to when the user reads it?